app/controllers/posts_controller.rb

```diff
@@ -0,0 +1,3 @@
+class PostsController < ApplicationController
+  def show; end
+end
```

Comment:
- [ ] [optional] @KareemWilson kindly add index method, its good practice to have a method that corresponds with the view eg a index page with index method